acceptance/localcluster/localcluster.go

```diff
@@ -40,6 +41,7 @@ const (
 var cockroachImage = flag.String("i", builderImage, "the docker image to run")
 var cockroachBinary = flag.String("b", defaultBinary(), "the binary to run (if image == "+builderImage+")")
 var cockroachEntry = flag.String("e", "", "the entry point for the image")
+var testCertsDir = filepath.Clean(os.ExpandEnv("${PWD}/../resource/test_certs"))
```

Comment:
`${PWD}` is funky here: this works for tests in the `acceptance` package but wouldn't work for localcluster's own tests (if it had any). Not sure it's worth fixing at the moment but we could have run.sh pass in the cert path.